js/counter.js

```diff
@@ -1,16 +1,97 @@
 var PLAYER = {
-  points: 0
+  points: 0,
+  teenagers: 0,
+  kettles: 0,
+  theaters: 0
 }
 
 function earnPoints() {
   PLAYER.points += 1
   updateUI()
 }
 
+function buyTeenagers(num) {
+  teenagerIncrease = () => {
+    PLAYER.teenagers += num
+    updateTeenagerDorm()
+  }
+  return teenagerIncrease
+}
+
+function buyKettles(num) {
+  kettleIncrease = () => {
+    PLAYER.kettles += num
+    updateKettleStorage()
+  }
+  return kettleIncrease
+}
+
+function buyTheaters(num) {
+  theaterIncrease = () => {
+    PLAYER.theaters += num
+    updateTheaterLocations()
+  }
+  return theaterIncrease
+}
```

Comment:
Since all of these functions do the same thing, they are ripe for some refactoring.